doc/sphinx/source/introduction.rst

```diff
@@ -14,102 +21,86 @@ Scenarios
-   end
-   }
-
-That's it, now it can be selected when starting new games.
+To add a new starting condition see :ref:`Starting Conditions <lua_tribes_tribes_start_conditions>`.
 
 Win conditions
 ^^^^^^^^^^^^^^
 
 In non player scenarios, win conditions define when one single player has won
 a game. The definitions of win conditions is very similar to defining
-initializations: We have to create a Lua script in
-``/data/scripting/win_conditions`` which returns an array with ``name``,
-``description`` and ``func``. Let's also make up a quick example: The first
-player to have 200 logs in his HQ wins the game. All others loose. Save the
-following file as ``/data/scripting/win_conditions/havest_logs.lua``.
+starting conditions which returns an array with ``name``, ``description`` and ``func``. 
+Let’s make up a quick example: The first player to have 200 logs in his HQ 
+wins the game. All others loose. Save the following file as 
+``data/scripting/win_conditions/harvest_logs.lua``.
 
 .. code-block:: lua
 
-   use("aux", "coroutine") -- for sleep
+   include "scripting/coroutine.lua"                              -- for sleep()
+   include "scripting/win_conditions/win_condition_functions.lua" -- for broadcast_objective()
 
    -- For translations, textdomain must be win_conditions
-   set_textdomain("win_conditions")
-
-   return {
-      name = _ "Harvest logs",
-      description = _ "The first player with 200 logs wins!",
-      func = function()
-         -- Find all valid players.
-         local plrs = wl.Game().players
-
-         -- Iterate all players, check if he is the winner
-         local winner = nil
-         local losers = {}
-         while not winner do
-            sleep(5000) -- we do this every 5 seconds
-            for idx,p in ipairs(plrs) do -- iterate the players array
-               local hq = p.starting_field.immovable
-               if hq:get_wares("log") >= 200 then
-                  -- We found the winner
-                  winner = p
-                  losers = plrs
-                  table.remove(losers,idx) -- Remove the winner from the losers table
+   local textdomain = "win_conditions"
+   push_textdomain(textdomain)
+
+   local wc_name = "Harvest logs"
+   -- This needs to be exactly like wc_name, but localized, because wc_name
+   -- will be used as the key to fetch the translation in C++
+   local wc_descname = _("Harvest logs")
+   local wc_version = 1
+   local wc_desc = _("The first player with 200 logs in his headquarter wins!")
+
+   local wc_definition = {                              -- The array defining the win condition
+      name = wc_name,
+      description = wc_desc,
+      peaceful_mode_allowed = true,
+      func = function()                                 -- The function processing the win condition
+         local plrs = wl.Game().players                 -- All players
+         broadcast_objective("win_condition",           -- Set objective with win condition for all players
```

Comment:
Sorry, i have trouble with github E-mails for some time, so i overlooked your answer here.

`peaceful_mode`: I thought also to explain this, but in the end i thought it is self explaining. A user who wants to dive into LUA has probably played a lot of games so he might know the meaning of this. I am open for a suggestion explaining this though

Having two examples shown here is a bit overkill, imho. The endless_game win condition can be opened by everyone so i do not feel the need to show it here again. And if the wincondition will be changed, the documentation might get outdated again because no one thinks about this.
But i will add a sentence that one can look at the endless_game win condition for an easy example and add explanations to some of the variables.

Reply:
What I meant was to explain the basic parameters in an explanatory text after the code example rather then in the code comments.
However this would be only the cherry on the cake and the current version is good to go as well.
So let's have it.